### DISCUSSION TITLE
post/jack-k8s-internals-part-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#95] [Announcements]
# post/jack-k8s-internals-part-2/

한 번쯤 내 팟이 뜰 때 내부적으로 어떻게 동작하는지 궁금하셨던 분들에게 제공하는 글입니다.

https://tech.kakaopay.com/post/jack-k8s-internals-part-2/

**Comment by SangHakLee** [comment]
👍👍👍
